packages/runtime/runtime-definitions/package.json

```diff
@@ -79,8 +79,13 @@
         "backCompat": false,
         "forwardCompat": false
       },
+      "InterfaceDeclaration_ISummarizerNode": {
```

Comment:
Can these interfaces be removed from package exports? Or marked as @internal?
There are two aspects to compat - compat across layers, but also not making breaking changes in minor releases. This goes to next, but theoretically, if these interfaces are used outside, we need to put a breaking.md note. I'd rather find a way to hide them and not care about problems like that.

Reply:
These are used across the FF repo so they need to be part of packages exports. What does marking them as @internal do? Does it remove them from being part of the public package? They will still be part of type validation as of now.